lib/client.js

```
@@ -1,149 +1,160 @@
-	useMultiFileAuthState,
-	isJidBroadcast
+  makeWASocket,
+  fetchLatestBaileysVersion,
+  makeCacheableSignalKeyStore,
+  DisconnectReason,
+  Browsers,
+  useMultiFileAuthState,
+  isJidBroadcast,
 } from 'baileys';
 import { ProxyAgent } from 'proxy-agent';
 import { EventEmitter } from 'events';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)